web/docs/guides/platform/logs.mdx

```diff
@@ -63,18 +51,13 @@ WHERE h.x_real_ip IS NOT NULL
 ![SELECT query example](/img/guides/platform/logs/select-query.png)
 
 
-You may notice that the table name reference, `edge_logs`, is required in order to formulate the SELECT query. Likewise, when querying the database logs, you would need to provide a table name reference as well.
-
-The table references are as follows:
-
-- API Edge - `edge_logs`
-- Database - `postgres_logs`
+The log source, `edge_logs`, is required in order to formulate the SELECT query. Likewise, when querying the database logs, you would need to provide a table name reference as well. The list of supported product sources can be found under the **Sources** dropdown.
```

Comment:
I'm not too sure what the these mean:

> The log source, `edge_logs`, is required in order to formulate the SELECT query. Likewise, when querying the database logs, you would need to provide a table name reference as well.

Perhaps just the last sentence is required

```suggestion
The list of supported product sources can be found under the **Sources** dropdown.
```